src/modules/routing/handlers/KeylistUpdateResponseHandler.ts

```diff
@@ -0,0 +1,10 @@
+import { Handler, HandlerInboundMessage } from '../../../agent/Handler'
+import { KeylistUpdateResponseMessage } from '../messages'
+
+export class KeylistUpdateResponseHandler implements Handler {
+  public supportedMessages = [KeylistUpdateResponseMessage]
+
+  public async handle(messageContext: HandlerInboundMessage<KeylistUpdateResponseHandler>) {
+    // TODO It should handle the response when agent calls `await this.consumerRoutingService.createRoute(connectionRecord.verkey)` and notify about the result.
```

Comment:
Why aren't we doing anything with the response? Are you going to address this in a separate PR?

Reply:
We think should do it in separate PR which would also add the step to wait for the response after calling `createRoute`. I didn't plan to add this at first, but I had to when I set `return_route` and update HTTP inbound transporter. I wasn't 100 % sure what to do here and it seemed kind of out of the scope of this PR.

I created #234 to address this.